contrib/ohi-release-notes/run.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 
 RT_PKG="github.com/newrelic/release-toolkit@latest"
 DICTIONARY_URL="https://raw.githubusercontent.com/newrelic/release-toolkit/v1/contrib/ohi-release-notes/rt-dictionary.yml"
-EXCLUDED_DEPENDENCIES_MANIFEST_URL="https://raw.githubusercontent.com/newrelic/release-toolkit/msanmiquel/exclude-dev-dependencies/contrib/ohi-release-notes/excluded-dependencies.yml"
+EXCLUDED_DEPENDENCIES_MANIFEST_URL="https://raw.githubusercontent.com/newrelic/release-toolkit/v1/contrib/ohi-release-notes/excluded-dependencies.yml"
```

Comment:
NIT: change it to a env var:
```suggestion
RELEASE_TOOLKIT_REFERENCE="${RELEASE_TOOLKIT_REFERENCE:=v1}"

EXCLUDED_DEPENDENCIES_MANIFEST_URL="https://raw.githubusercontent.com/newrelic/release-toolkit/${RELEASE_TOOLKIT_REFERENCE}/contrib/ohi-release-notes/excluded-dependencies.yml"
```